source/reference/beta-program/orr.txt

```diff
@@ -13,3 +13,53 @@ Enable Oplog Rollover Resilience
    :class: singlecol
 
 .. include:: /includes/beta-feature.rst
+
+.. include:: /includes/orr-intro.rst
+
+By applying changes earlier in the sync process, ``mongosync`` maintains a more 
+recent position in the :term:`oplog`. This adds resilience to long-running 
```

Comment:
(question)
Is "a more recent position" a jargon-y phrase? Is there a different way to phrase this? I am less familiar with C2C, so I'm genuinely asking. : )

Reply:
Gotcha — "more recent position" refers to how the `mongosync` operation is recorded higher up in the oplog (hence in a more recent position)! Can't say "most recent operation" since there are likely other change events being logged by the oplog, which is why I ended up with "more recent position."

Reply:
Ok, got it! Thanks for the explanation. That works for me!